accounts/views.py

```diff
@@ -4,7 +4,20 @@
 from django.contrib.auth.decorators import login_required
 
 
+def login_excluded(redirect_to):
+    """ Kicks authenticated users out of a view """
+    def _method_wrapper(view_method):
```

Comment:
why do you need function in function in function? 
I think you need to create a class here instead

Reply:
When I looked up how to do it the results always had functions in functions, didn't find much about making it a class and so far I haven't managed to get it to work as a class. Should I remove it and try to find another way to do this?

Reply:
I think it's not clear like that 
@guy9050 what do you think?

Reply:
agree, @mayasa3 I see that you deleted it.. do you not need it eventually?

Reply:
I didn't delete it, but I moved it to a different file so that it(and other custom decorators if anyone needs/wants to add them) won't be limited to one app. 
Right now I mostly just need to keep logged in users out of these pages, I can try a different way but I thought making it similar to @login_required was a better idea at the time

Reply:
So please take a look in other projects
you can't keep it like that

Reply:
Deleted it now

Reply:
Does it work?

Reply:
@mayasa3 ?

Reply:
It works on my end

Reply:
Works on my end but I'll double check to make sure

Reply:
I changed sign up to a class-based view while I was at it, everything still works so far

Reply:
@bardielle @guy9050 does it look ok now?

Reply:
Yes